doc/rst/technotes/gpu.rst

```diff
@@ -121,12 +121,15 @@ Requirements
     version as the bundled version (currently 14). Older versions may
     work; however, we only make efforts to test GPU support with this version.
 
-* Either ``nvcc`` (for NVIDIA) or ``hipcc`` (for AMD) must be available; Chapel
-  uses libraries included in these packages and will automatically deduce the
-  path to these libraries based on the location of the ``nvcc``/``hipcc``
-  executable. Note that the automatically deduced paths may be overwritten by
-  manually setting the ``CHPL_CUDA_PATH`` or ``CHPL_ROCM_PATH`` environment
-  variables.
+* Either the CUDA toolkit (for NVIDIA), or ROCM (for AMD) must be installed.
+
+  * If targeting NVIDIA GPUs, we require CUDA toolkit to be version 10.x or 11.x
+    (inclusive). If using version 10.x you must set
+    ``CHPL_RT_NUM_THREADS_PER_LOCALE=1``. In some cases versions as early as
+    7.x or at or above 12.x may work but we have not tested these.
+
+  * If targeting AMD GPUs, we require a version of ROCM greater than 4.x.
```

Comment:
OOC, did you try with ROCM 5.x? If we are uncertain, we can talk more strictly about 4.x here.

Reply:
No, as of today I've only tested it with 4.x.

I wrote it to suggest that 5.x might work because my plan in my other PR (https://github.com/chapel-lang/chapel/pull/22093) was to error out on versions < 4.x but otherwise don't complain.

I'd rather not lock someone out of using a future version unless we're certain there's an issue with it (although I could be convinced otherwise).

How about I rewrite the sentence to:

"If targeting AMD GPUs, we require ROCM version 4.x; we suspect version 5.x will work as well although we have not tested so."

Reply:
Yeah, I think I prefer that one.